contributing.md

```diff
@@ -0,0 +1,72 @@
+# Addendum to C&C WG Work Item: Credential Presentation
+
+[NON-NORMATIVE/NON-BINDING] Guidelines for Repository Process
+
+NOTE TO EDITORS (delete Friday 30/7/20): this was taken from the Repo guidelines attached to the DIDComm working group charter found [here](https://drive.google.com/file/d/1gkzkEUw1oa8vECorlLBSRCp2vgT7aqmh/view). The analogy may break down since that was a working group addendum signed by all companies to amend the charter of the working group, whereas this is an inherently non-normative document applying only to a specific spec work item, to see it through to a v0.1.0 and beyond. 
+
+# Working Group Repository Process
+
+The Decentralized Identity Foundation (DIF) provides guidelines for Repository Process that each Working Group can "fork," fine-tune, and ratify during their meetings. The process is designed to simplify and consolidate GitHub status tracking of work items managed by the Working Group. 
+
+## Repository Process
+
+### Repository Maintainer 
+Maintainers are expected to handle the repository management as described in the Repository Process (below).
+
+### Repository Maintainer selection
+Maintainers must be Working Group Participants selected by Working Group Participants via consensus. At 	least 3 maintainers must be to be active at any given time operating the Repository Process for the Working Group. Maintainers *MUST* have different organizational affiliations to represent the diversity of the Working Group (i.e., no two maintainers should work for the same company, organization, or project).
+
+### Specification Formatting Requirements
+
+* 80-char fixed-width columns (aka "hard wraps") required by this specification.
+
+### Repository process
+
+The Repository process is an extension of the *Deliverable Development Process – Draft* section detailed in DIF’s Charter. The goal of this process is to introduce a repository management processes for the work section developed by the Working Group. 
```

Comment:
```suggestion
The Repository process is an extension of the *Deliverable Development
Process – Draft* section detailed in DIF’s Charter. The goal of this process
is to introduce a repository management processes for the work section
developed by the Working Group. 
```
In light of the recommended formatting requirements, should this document be hard wrapped at 80 characters?